### ISSUE TITLE
Blurred face

### ISSUE THREAD

**Issue by lubovski** [Gourieff/comfyui-reactor-node#496]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/comfyui-reactor-node/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

Almost all my attempts at face replacement lead to blurry results. I tried different faces, different detection and restoration models - the result is the same. This problem is also reproduced on the Tensor Art site, so I rule out a problem with my equipment.
![Screenshot_1](https://github.com/user-attachments/assets/469a90a1-328e-49df-b901-15fa904cc5b6)

### Steps to reproduce the problem

Image generation - face replacement

### Sysinfo

Windows 10, Firefox, RTX 3070

### Relevant console log

```Shell
Loading 1 new model
loaded completely 0.0 159.55708122253418 True
[ReActor] 02:32:48 - STATUS - Working: source face index [0], target face index [0]
[ReActor] 02:32:48 - STATUS - Analyzing Source Image...
[ReActor] 02:32:50 - STATUS - Trying to halve 'det_size' parameter
[ReActor] 02:32:51 - STATUS - Analyzing Target Image...
[ReActor] 02:32:53 - STATUS - Swapping...
[ReActor] 02:33:01 - STATUS - --Done!--
[ReActor] 02:33:01 - STATUS - Restoring with GFPGANv1.4.pth | Face Size is set to 512
Warning torch.load doesn't support weights_only on this pytorch version, loading unsafely.
        Starting restore_face with codeformer_fidelity: 0.1

0: 640x448 1 face, 145.7ms
Speed: 4.0ms preprocess, 145.7ms inference, 55.3ms postprocess per image at shape (1, 3, 640, 448)
Prompt executed in 53.53 seconds
```

**Comment by Gourieff** [comment]
Show your workflow please and provide your input and the source images as well
If a face in the input image is too large dimension - the restoration model won't restore it

**Comment by lubovski** [comment]
> Show your workflow please and provide your input and the source images as well If a face in the input image is too large dimension - the restoration model won't restore it

I used different faces - big and small. Here is an example of a photo of a face, the total resolution is 726x402, and the face in the photo takes up a small part.

![1](https://github.com/user-attachments/assets/003b731d-d3fe-4134-b25e-3f914f15dcbb)
![2](https://github.com/user-attachments/assets/e78115e3-77df-47e2-9c93-312b21b887b8)
![Face](https://github.com/user-attachments/assets/d58870dc-39bd-4043-bbf2-70e26da30d2f)
[WF.json](https://github.com/user-attachments/files/17936882/WF.json)

**Comment by Gourieff** [comment]
You probably do smth wrong, you don't need to crop the face before swap or to restore it after with additional restorers
Just open any of the images below in your comfy to load the WF, it's rather simple to swap even with large pictures
![+ComfyUI_temp_vyrhc_00040_](https://github.com/user-attachments/assets/93a09c44-84b8-4cb5-a3f9-e8a936ecdede)
![+ComfyUI_temp_vyrhc_00038_](https://github.com/user-attachments/assets/b61de147-cec7-41d3-a9f4-650806fa1af5)

**Comment by lubovski** [comment]
Yes, your workflow really works. I will try to figure out what is the problem in my workflow. Thank you very much!

**Comment by Imageman** [comment]
![изображение](https://github.com/user-attachments/assets/a49560bd-f36f-45bd-882e-538f7e13763f)
I am not succeeding. The face in your process also becomes pixelated. Perhaps my neural network is outdated? Boost only partially helps. Faces below a certain threshold don't have such issues.
